### ISSUE TITLE
setDefaults is not defined when passing options

### ISSUE THREAD

**Issue by hvaughan3** [clampy-js/vue-clampy#2]
Attempting to pass options when adding the plugin to the global Vue instance produces the following exceptions:

```
Uncaught ReferenceError: setDefaults is not defined
    at Object.install (vue-clampy.js?6bc5:374)
    at Function.Vue.use (vue.esm.js?efeb:4751)
    at eval (main.js?3479:46)
    at Object../src/main.js (app.js:10469)
    at __webpack_require__ (app.js:679)
    at fn (app.js:89)
    at Object.1 (app.js:10750)
    at __webpack_require__ (app.js:679)
    at app.js:725
    at app.js:728
```

Code:

    import clampy from '@clampy-js/vue-clampy'

    Vue.use(clampy, {
      truncationChar: '&nbsp;'
    })

Looking at the code, there is indeed `setDefaults` [method call](https://github.com/clampy-js/vue-clampy/blob/931557848b9e2e61677a1477f086e6742f7f6fb6/vue-clampy.js#L374) but I do not see a `setDefaults` method defined anywhere.

**Comment by alexandremoore** [comment]
@hvaughan3 , can you try with 1.0.2?

**Comment by alexandremoore** [comment]
@hvaughan3, this is now fixed in 1.0.4. Feel free to reopen the issue if necessary.

**Comment by hvaughan3** [comment]
Very sorry for the delay. Just tried it out and it does not crash but I do not see the options actually taking effect when passing them through the `Vue.use()` line.
